importer/src/test/java/org/hiero/mirror/importer/parser/domain/RecordFileBuilder.java

```diff
@@ -118,9 +119,13 @@ public class ItemBuilder {
 
         private int count = 100;
         private int entities = 10;
+        private int nonce = 0;
         private boolean entityAutoCreation = false;
+        private boolean isScheduled = false;
```

Comment:
Lombok should generate the getter as `isScheduled()`
```suggestion
        private boolean scheduled = false;
```